### ISSUE TITLE
Issue with allControls and forceSelect

### ISSUE THREAD

**Issue by sebage** [js-soft/wdi5#225]
I am using **browser.allControls** in combination with **forceSelect** to fetch multiple instances of sap.m.CheckBox. Afterwarts i loop over the result and call press() on the selected checkboxes. This works without forceSelect, but as soon as i use forceSelect every press() in the loop is performed on the first checkbox only. The checkboxes are within a sap.m.Table in a Dialog. Using the latest RC4 version. See coding below:

```
   oFilterDialogCheckboxSelector: {
          forceSelect: true,
          selector: {
              controlType: "sap.m.CheckBox",
              viewName: sMasterViewName,
              properties: {
                  editable: true
              },
              searchOpenDialogs: true,
          }
    }
...
    async iDeselectAllFilters() {
        let aFilterCheckbox = await browser.allControls(oMasterSelectors.oFilterDialogCheckboxSelector);

        for ( let oCheckbox of aFilterCheckbox) {
                if (await oCheckbox.getSelected()) {
                    await oCheckbox.press();
                }            
        }
    }
```

**Comment by dominikfeininger** [comment]
yep, you are right. Working on it: https://github.com/js-soft/wdi5/pull/226

**Comment by vobu** [comment]
FTR: successfully published wdio-ui5-service@0.9.0-rc4.1
